### PR TITLE
Get.md

### PR DESCRIPTION
**Pull request by delas6006**
get.md repair

### CODE REVIEW

File: get.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+###GET /meal
```
Comment (Laura-Novich-OBW) [inline get.md:1]:
```suggestion
# GET /meal
```

File: get.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+##HTTP Responses
```
Comment (Laura-Novich-OBW) [inline get.md:3]:
```suggestion
## HTTP Responses
```

File: get.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+#Status Type Code 
+| Status            | Type   | Code Description          |
+|-------------------|--------|---------------------------|
+| 200 - OK          | string | Order is correct.         |
+| 400 - Bad Request | string | Meal ordered is incorrect |
+
+GET Retrieve the currently available meal Retrieves the meal that is available at the current time.
```
Comment (Laura-Novich-OBW) [inline get.md:11]:
```suggestion
This call retrieves the currently available meal Retrieves the meal that is available at the current time.
```

File: get.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+#GET /meal
```
Comment (Laura-Novich-OBW) [inline get.md:13]:
```suggestion
## GET /meal Parameters
```

File: get.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Arguments (Query Parameters) There are no arguments.
+
+#Response Parameters
```
Comment (Laura-Novich-OBW) [inline get.md:17]:
```suggestion
# Response Parameters
```

File: get.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|
```
Comment (Laura-Novich-OBW) [inline get.md:22]:
```suggestion
                                                                    
```

File: get.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+Arguments (Query Parameters) There are no arguments.
+
+#Response Parameters
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  |
```
Comment (Laura-Novich-OBW) [inline get.md:23]:
```suggestion

```

File: get.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+Arguments (Query Parameters) There are no arguments.
+
+#Response Parameters
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S
```
Comment (Laura-Novich-OBW) [inline get.md:24]:
```suggestion
| PattySize  | string | Required | L, M, or S 
```

File: get.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+#Response Parameters
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    |
```
Comment (Laura-Novich-OBW) [inline get.md:25]:
```suggestion
| BurgerCook | string | Required | LE, Phil                                    | 
```

File: get.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+#Response Parameters
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           |
```
Comment (Laura-Novich-OBW) [inline get.md:26]:
```suggestion
| BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
```

File: get.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
+| | Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            |
```
Comment (Laura-Novich-OBW) [inline get.md:27]:
```suggestion
| Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            | 
```

File: get.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
+| | Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            | 
+| | FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” |
```
Comment (Laura-Novich-OBW) [inline get.md:28]:
```suggestion
| FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” | 
```

Comment (Laura-Novich-OBW) [reply get.md:28]:
```suggestion
| FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” | 
```

File: get.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+| #Parameter   |#Data type| #Required? |#Description |
+|--------------------------------------------------------------------------------
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
+| | Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            | 
+| | FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” | 
+| | DrinkType  | string | Required | Coke, Sprite, Sprite Zero, “Ice Tea”        |
```
Comment (Laura-Novich-OBW) [inline get.md:29]:
```suggestion
| DrinkType  | string | Required | Coke, Sprite, Sprite Zero, “Ice Tea”        | 
```

File: get.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
+| | Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            | 
+| | FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” | 
+| | DrinkType  | string | Required | Coke, Sprite, Sprite Zero, “Ice Tea”        | 
+
+curl -X GET http://URL/tableNo?id=99
+
+```{
```
Comment (Laura-Novich-OBW) [inline get.md:33]:
```suggestion
``` json
 { 
```

File: get.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+|                                                                                  
+|                                  | 
+| | PattySize  | string | Required | L, M, or S 
+| | BurgerCook | string | Required | LE, Phil                                    | 
+| | BunType    | string | Required | Seeded, Regular, or glutenFreeBun           | 
+| | Fixing1    | string | Required | Lettuce, Tomato, Onion or “None”            | 
+| | FriesType  | string | Required | RegularFries, “ SweetPotatoFries” or “None” | 
+| | DrinkType  | string | Required | Coke, Sprite, Sprite Zero, “Ice Tea”        | 
+
+curl -X GET http://URL/tableNo?id=99
```
Comment (Laura-Novich-OBW) [inline get.md:31]:
```suggestion
``` curl
      curl -X GET http://URL/tableNo?id=99
```

File: get.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+"BunType":"Regular", 
+"BurgerCook":"LE", 
+"Fixing1":"Lettuce",”Tomato”,”Onion”, 
+“Cost”:35.00 } },
+
+}
+ "Total":{ "Cost":35.00 }
+
+ }
+ }```
```
Comment (Laura-Novich-OBW) [inline get.md:49]:
```suggestion
 } ```
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
please approve or reject suggestions and commit.

**Comment by delas6006** [comment]
Completed the corrections

File: get.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
+# GET /meal
+
+## HTTP Responses
+
+#Status Type Code
```
Comment (Laura-Novich-OBW) [inline get.md:5]:
```suggestion
## Status Type Code 
```

**Pull request merged**